bin/cli

```diff
@@ -1,23 +1,38 @@
 #!/usr/bin/env node
 
-// Perform rewrites to the runtime environment variables based on the contents
-// of the process.env.REWRITE_ENV if it exists. This is done here as it is the
-// entrypoint for the entire application.
-require('env-rewrite').rewrite();
-
 /**
  * Module dependencies.
  */
 
 const program = require('commander');
 const pkg = require('../package.json');
+const dotenv = require('dotenv');
```

Comment:
I think it does offer benift as it removes the syntax of the:

```
export NAME=value
```

which to me shouldn't exist in a config file. The config flag, if you noticed, specifically loads the config from a named file, this way a user could specify another file to load config from instead.